adapters/repos/db/inverted/objects_test.go

```diff
@@ -515,6 +515,59 @@ func TestAnalyzeObject(t *testing.T) {
 			assert.ElementsMatch(t, expectedNumbers, actualNumbers, res)
 			assert.ElementsMatch(t, expectedUUID, actualUUID, res)
 		})
+
+		t.Run("when rep prop is stored as empty string slice", func(t *testing.T) {
```

Comment:
I don't fully get what you're testing here, could you add a little explanation? Would also help in the future when revisiting the test

Reply:
sure. added